src/ontology/mondo-edit.obo

```diff
@@ -430439,6 +430582,410 @@ property_value: IAO:0000231 "out of scope" xsd:string
+def: "A rare disorder due to poisoning characterized by acute onset of potentially life-threatening illness following ingestion, inhalation, or injection of ricin, a lectin present in the seeds of <i>Ricinus communis</i>, the castor oil plant. Clinical presentation depends on the route of administration, inhalation being the most toxic route, followed by oral ingestion. Presenting signs and symptoms include nausea, vomiting, diarrhea, hematemesis, and melena (upon ingestion), cough, wheezing, dyspnea, sore throat, and congestion (upon inhalation), and erythema, induration, blisters, capillary leak syndrome, and localized necrosis (upon injection). The condition can progress to seizures, shock, organ failure, pulmonary edema, and respiratory failure." [Orphanet:570470]
+subset: ordo_disorder
+xref: ICD10:T62.2 {source="Orphanet:570470", source="MONDO:mondoSubClassOfSource"}
+xref: Orphanet:570470 {source="MONDO:equivalentTo"}
+is_a: MONDO:0029000 {source="https://orcid.org/0000-0001-5208-3432"} ! poisoning
+
+[Term]
+id: MONDO:0035529
+name: infantile-onset pulmonary alveolar proteinosis-hypogammaglobulinemia
+def: "A rare genetic respiratory disease characterized by infantile onset of pulmonary alveolar proteinosis with hypogammaglobulinemia. Patients have normal respiratory function at birth, but subsequently develop recurrent, mainly viral, infections and progressive respiratory failure, often leading to death in infancy or early childhood. Additional reported features include leukocytosis and splenomegaly." [Orphanet:572428]
+subset: ordo_disorder
+synonym: "OAS1 deficiency" EXACT [Orphanet:572428]
+synonym: "OAS1-related infantile-onset pulmonary alveolar proteinosis-hypogammaglobulinemia" EXACT [Orphanet:572428]
+xref: Orphanet:572428 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005087 {source="https://orcid.org/0000-0001-5208-3432"} ! respiratory system disorder
+relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/8086 ! OAS1
+
+[Term]
+id: MONDO:0035534
+name: DONSON-related microcephaly-short stature-limb abnormalities spectrum
+def: "A rare autosomal recessive microcephalic primordial dwarfism characterized by congenital microcephaly and craniofacial features associated with a spectrum of limb abnormalities ranging from mild to severe. Short stature is frequently observed and often is severe." [Orphanet:572761]
+subset: ordo_disorder
+xref: Orphanet:572761 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! Mendelian disease
+
+[Term]
+id: MONDO:0035541
+name: split cord malformation type II
+subset: ordo_subtype_of_a_disorder
+synonym: "SCM type 2" EXACT [Orphanet:573253]
+synonym: "SCM type II" EXACT [Orphanet:573253]
+synonym: "split cord malformation type 2" EXACT [Orphanet:573253]
+xref: Orphanet:573253 {source="MONDO:equivalentTo"}
+is_a: MONDO:0002320 {source="https://orcid.org/0000-0001-5208-3432"} ! congenital nervous system disorder
+
```

Comment:
note that this might be a phenotype: 
from: https://pubmed.ncbi.nlm.nih.gov/1407428/
"A Type II SCM consists of two hemicords housed in a single dural tube separated by a nonrigid, fibrous median septum."

Reply:
ah good point, I'll remove this